src/components/TabSelector/TabSelector.tsx

```diff
@@ -13,7 +13,7 @@ import CONST from '@src/CONST';
 import type IconAsset from '@src/types/utils/IconAsset';
 import TabSelectorItem from './TabSelectorItem';
 
-type TabSelectorProps = {
+export type TabSelectorProps = {
```

Comment:
We don't use inline exports, please add export to the bottom of this file

Reply:
Hey blazekustra. Ok I'll fix it.